azurerm/internal/services/applicationinsights/client/client.go

```diff
@@ -10,6 +10,7 @@ type Client struct {
 	APIKeysClient        *insights.APIKeysClient
 	ComponentsClient     *insights.ComponentsClient
 	WebTestsClient       *insights.WebTestsClient
+	BillingFeatureClient *insights.ComponentCurrentBillingFeaturesClient
```

Comment:
Could we just call this
```suggestion
	BillingClient *insights.ComponentCurrentBillingFeaturesClient
```

Reply:
Renamed from BillingFeatureClient to BillingClient.